index.bs

```diff
@@ -5796,6 +5800,41 @@ At this time, one [=credential property=] is defined: the [=resident key credent
 
 For the purposes of user agent automation and [=web application=] testing, this document defines a number of [[WebDriver]] [=extension commands=].
 
+## WebAuthn WebDriver Capability ## {#sctn-automation-webdriver-capability}
```

Comment:
```suggestion
## WebAuthn WebDriver Extension Capability ## {#sctn-automation-webdriver-capability}
```

Reply:
Done.

Reply:
huh, it seems line 5803 wasn't changed?

Reply:
I think you might be looking at the outdated patch (if you see above there's an `outdated` label next to the filename).

Reply:
oh shucks, sorry, I meant to remove that comment and the "ditto" below. yes was looking at outdated stuff for some reason.